apis/python/src/tiledb/vector_search/ivf_flat_index.py

```diff
@@ -495,7 +507,7 @@ def create(
             centroids_array_rows_dim, centroids_array_cols_dim
         )
         centroids_attr = tiledb.Attr(
-            name="centroids",
+            name="values",
```

Comment:
Could this be the issue?

Reply:
We need to make sure that this is backwards compatible and the new code can read old assets